src/test/java/de/tonypsilon/bmm/backend/referee/service/RefereeServiceTest.java

```diff
@@ -238,7 +246,7 @@ void testFindById() {
         when(refereeRepository.findById(2L)).thenReturn(Optional.empty());
         Optional<RefereeData> actual = refereeService.findById(1L);
         assertTrue(actual.isPresent());
```

Comment:
assertTrue and the current assertion can be chained using AssertJ's OptionalAssert -> .isPresent() followed by .hasValue()